src/Infrastructure/Swagger/AddTenantIdProcessor.cs

```diff
@@ -27,9 +27,7 @@ public bool Process(OperationProcessorContext context)
                 Kind = OpenApiParameterKind.Header,
                 Description = attribute.Description,
                 IsRequired = attribute.IsRequired,
-                Schema = string.IsNullOrEmpty(attribute.DefaultValue)
-                    ? null
-                    : new NJsonSchema.JsonSchema
+                Schema = new NJsonSchema.JsonSchema
```

Comment:
why is this null check removed?

Reply:
Hmm... It's probably ok... I thought Schema.Default was not nullable... but the lib is probably not yet "nullable aware".

Reply:
the tool to generate the proxy classes doesn't understand and generates a signature with a parameter of type object (generic)